### PR TITLE
Added section on ownership of flags in environments

### PR DESCRIPTION
**Pull request by trmartin4**
## 🎟️ Tracking

Feedback from team members on lack of understanding of feature flag ownership between environments

## 📔 Objective

As we become more mature in our flag usage and have a USDEV environment, I felt it would be helpful to clarify what the expectations are for flag usage in each environment.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation
  team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

**Comment by trmartin4** [comment]
@withinfocus when looking at this section, I encountered this text that may no longer be best practice:

>Feature flags don’t necessarily have to ever be deleted from LaunchDarkly, just unused. Linking them
to Jira helps create a history of the feature and there are copious logs and audit records online
that can be kept.

Is this still our recommendation?  I believe we now recommend that we should delete from LD, right?

File: docs/contributing/feature-flags.md [hunk 1]
```diff
@@ -254,10 +254,54 @@ the retrieval methods:
...
+:::tip LaunchDarkly access
+
 Let your management know when you need to change something about a feature online inside
 LaunchDarkly. Only a small number of users have accounts with LaunchDarkly to save on licensing
 costs.
 
+:::
+
+The process for updating flag values and the ownership of flag changes depends on the environment
+and change as a flag proceeds through our cloud landscape.
```
Comment (withinfocus) [inline docs/contributing/feature-flags.md:266]: ⛏️ "cloud landscape" sounds beautiful but perhaps too whimsical for docs 😜

File: docs/contributing/feature-flags.md [hunk 2]
```diff
@@ -254,10 +254,54 @@ the retrieval methods:
...
+
+### Development environments
+
+In the development environments, ownership of the flag values is in the hands of the development
+team. The Engineering Manager of the team is responsible for maintaining and updating flag values.
+
+By default, flags should be **enabled** in the development environment as soon as the flag is
+created. This allows the feature to be viewed and exercised by Product, Design, the development
```
Comment (withinfocus) [inline docs/contributing/feature-flags.md:274]: ⛏️ I don't consider product or design groups to be proper nouns.

File: docs/contributing/feature-flags.md [hunk 3]
```diff
@@ -254,10 +254,54 @@ the retrieval methods:
...
+the feature is `Ready for QA` the QA Engineers on the team should work with the QA Managers to
+enable the flag.
+
+The QA Managers are also responsible for ensuring that during regression testing the environment
+under test has the proper flag values set to match the desired production environment after
+deployment.
+
+### Production environments
+
+In the production environments, the ownership of the flag values is shared between Product and
```
Comment (withinfocus) [inline docs/contributing/feature-flags.md:297]: ⛏️ Same.

**Comment by withinfocus** [comment]
> @withinfocus when looking at this section, I encountered this text that may no longer be best practice:
> 
> > Feature flags don’t necessarily have to ever be deleted from LaunchDarkly, just unused. Linking them
> > to Jira helps create a history of the feature and there are copious logs and audit records online
> > that can be kept.
> 
> Is this still our recommendation? I believe we now recommend that we should delete from LD, right?

Huh, I don't remember this language being so explicit. We archive flags, not delete them, so maybe that was more the intention.

File: docs/contributing/feature-flags.md (hunk 1 above)
Comment (trmartin4) [reply docs/contributing/feature-flags.md]: Updated in [`151b341` (#389)](https://github.com/bitwarden/contributing-docs/pull/389/commits/151b341f9f5d78b41d6a78d3315ed79f0659e45d) to be a bit less poetic.

File: docs/contributing/feature-flags.md (hunk 2 above)
Comment (trmartin4) [reply docs/contributing/feature-flags.md]: Fixed in [`151b341` (#389)](https://github.com/bitwarden/contributing-docs/pull/389/commits/151b341f9f5d78b41d6a78d3315ed79f0659e45d).

File: docs/contributing/feature-flags.md (hunk 3 above)
Comment (trmartin4) [reply docs/contributing/feature-flags.md]: [`151b341` (#389)](https://github.com/bitwarden/contributing-docs/pull/389/commits/151b341f9f5d78b41d6a78d3315ed79f0659e45d)

**Comment by trmartin4** [comment]
> > @withinfocus when looking at this section, I encountered this text that may no longer be best practice:
> > > Feature flags don’t necessarily have to ever be deleted from LaunchDarkly, just unused. Linking them
> > > to Jira helps create a history of the feature and there are copious logs and audit records online
> > > that can be kept.
> > 
> > 
> > Is this still our recommendation? I believe we now recommend that we should delete from LD, right?
> 
> Huh, I don't remember this language being so explicit. We archive flags, not delete them, so maybe that was more the intention.

I removed that language and instead referenced the archive process at the end of the "Unwinding a feature flag" section.

**Review by withinfocus — APPROVED** [review]

**Comment by withinfocus** [comment]
I think Todd is out so I am gonna merge this.

**Pull request merged**